Gemfile

```diff
@@ -1,2 +1,6 @@
 source 'https://rubygems.org'
 gemspec
+gem 'pry-rails'
```

Comment:
It would be better to define the `pry` dependency in the `gemspec` file, since it is called here in line 2. Please add the following line there: `spec.add_development_dependency 'pry', '~> 0.10.3'`